src/graphqlClient.ts

```diff
@@ -77,7 +78,7 @@ export class GraphqlClient {
 
         const query = gql`
             query getDatabaseClients {
-                databaseClients {
```

Comment:
I know this is demo-level, but maybe there should be a note in readme that this script works only if you have fewer than 1000 clients.